tools/Azure.Mcp.Tools.Foundry/src/Options/Models/KnowledgeIndexSchemaOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using Azure.Mcp.Core.Options;
+
+namespace Azure.Mcp.Tools.Foundry.Options.Models;
+
+public class KnowledgeIndexSchemaOptions : GlobalOptions
+{
+    [JsonPropertyName(FoundryOptionDefinitions.Endpoint)]
+    public string? Endpoint { get; set; }
```

Comment:
Do all index-related commands need `Endpoint`? If so, should work be done to create `BaseKnowledgeIndexOptions` so we don't need to repeat endpoint in more places.